### PR TITLE
multi: add more trace logs regarding link activate flow

### PR DESCRIPTION
**Pull request by yyforyongyu**
This PR adds more trace logs to aid debugging.

### CODE REVIEW

File: peer/brontide.go [hunk 1]
```diff
@@ -776,7 +776,8 @@ func (p *Brontide) loadActiveChannels(chans []*channeldb.OpenChannel) (
 
 		chanID := lnwire.NewChanIDFromOutPoint(chanPoint)
 
-		p.log.Infof("loading ChannelPoint(%v)", chanPoint)
+		p.log.Infof("loading ChannelPoint(%v), isPending=%v",
```
Comment (guggero) [inline peer/brontide.go:779]: nit: begin log message uppercase?

**Review by guggero — APPROVED** [review]
LGTM :tada:

File: peer/brontide.go (hunk 1 above)
Comment (yyforyongyu) [reply peer/brontide.go]: good catch! done!

**Review by bitromortac — APPROVED** [review]
LGTM :zap:

**Pull request merged**